test-requirements.txt

```diff
@@ -7,4 +7,5 @@ boto3>=1.10.45
 moto[all]>=1.3.14
 smart-open>=5.0.0
 pyarrow>=3.0.0
-alteryx-open-src-update-checker>=2.0.0
\ No newline at end of file
+alteryx-open-src-update-checker>=2.0.0
+pyorc>=0.4.0
```

Comment:
Do we need this dependency?

Reply:
No - removing that and fixing a couple other things now...